app.libretile.Libretile.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: app.libretile.Libretile
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: libretile.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+build-options:
+    build-args:
+      -    --share=network
+modules:
+  - name: libresprite
+    buildsystem: simple
+    
+    build-commands:
+       - pip3 install --prefix=/app pillow
+       - git clone https://gitlab.com/celleron56/wavefunctioncollabseterrain 
+       - mv wavefunctioncollabseterrain /app/files
+       - cd /app/files
+       - echo "python3  /app/files/main.py" >> libretile.sh
+       - ln -s libretile.sh libretilelink.sh
+       - install -D libretilelink.sh /app/bin/libretile.sh
+    sources:
+        type: git
+        path: /app/files
+        url: https://gitlab.com/celleron56/wavefunctioncollabseterrain
```

Comment:
this is not how it works. You can't set `path` to the destination. `path` is related to the build dir

Reply:
ok will fix it thx